artifacts/deploy/cluster-proxy-admin-rbac.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
We can add a description of the purpose in the header of the file.
Mark it as optional

Reply:
Is following description ok?
```
# This configuration is used to authorize system:admin to proxy member clusters,
# if you don't need it, you can remove it from karmada control plane.

apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: cluster-proxy-admin
......
```

Reply:
+1